src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareOperation.cs

```diff
@@ -120,8 +120,9 @@ public void Execute(TaskExecutionMode mode)
                     }
                 }
 
-                // Appending the set of errors that are stopping the schema compare to the ErrorMessage 
-                var errorsList = ComparisonResult.GetErrors().Select(e => e.Message).Distinct().ToList();
+                // Appending the set of errors that are stopping the schema compare to the ErrorMessage
+                // GetErrors return all type of warnings, and error messages. Only filtering the error type messages here
+                var errorsList = ComparisonResult.GetErrors().Where(x => x.MessageType.Equals(Microsoft.SqlServer.Dac.DacMessageType.Error)).Select(e => e.Message).Distinct().ToList();
```

Comment:
can you add a check in the schema compare tests to check that the expected value of the error message?

Reply:
+ A test to see if we are getting errors only when that result.success is false or otherwise. 

Please try to update the dlls in ADS and run the integration tests for ADS also. Let us know for any help needed there.